src/main/java/frc/robot/commands/GamePieaceManipulator_Score.java

```diff
@@ -0,0 +1,29 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.subsystems.GamePieceManipulatorSubsystem;
+
+/**
+ * A GamePieceManipulatorCommands uses the GamepieceMainpulatorSubsytems
+ */
+public class GamePieaceManipulator_Score extends Command {
```

Comment:
Same comments as the GamePieceManipulator_Eject command. A good name for this might be ScoreCoralCommand. We might end up having a separate command for each level on the reef anyways.